app/uk/gov/hmrc/helptosavestub/controllers/UserInfoAPIController.scala

```diff
@@ -68,8 +69,11 @@ class UserInfoAPIController extends BaseController {
         val message = "Could not find Authorization in header"
         Logger.error(message)
         Unauthorized(message)
-      } { token ⇒
-        userInfoGen.seeded(token).fold(
+      }{ token ⇒
+        val headerToken = headers.get("token")
```

Comment:
The token doesn't come from the auth-login-stub. Normally, the user logs in through the auth-login-stub, then our frontend tries to retrieve an authorsation code from oauth-frontned. There's a flag in the frontend to skip communicating with oauth-frontend and just use the NINO as the authorisation code. The authorisation code is then exchanged for a token using the `OAuthController` in our stub, and it is this token which is used to call the `UserInfoAPIController` here